docs/release-notes/next-minerva/PT-162792344-new-aevm-version.md

```diff
@@ -0,0 +1,2 @@
+* Introduces a new AEVM version to contain consensus breaking changes and optimizations.
+* Removes references to old planned VM versions that will not be implemented.
```

Comment:
This seems not visible to the user so unnecessary.

Reply:
Since this is visible in the Protocol (where I will make a PR as well), I thought I'd mention it here for completeness.